tests/behaviour/concept/thing/relation/relation_steps.py

```diff
@@ -87,51 +87,52 @@ def step_impl(context: Context, var1: str, role_label: str, var2: str):
 def adding_player_throws_exception(context: Context, var1: str, role_label: str, var2: str):
     relation = context.get(var1).as_relation()
     try:
-        relation.as_remote(context.tx()).add_player(
-            relation.get_type().as_remote(context.tx()).get_relates(role_label),
+        relation.add_player(
+            context.tx(),
+            relation.get_type().get_relates(context.tx(), role_label),
             context.get(var2))
         assert False;
-    except TypeDBClientException:
+    except (TypeDBClientException, RuntimeError):
```

Comment:
Ideally the user would only have to catch 1 exception, instead of 2. A "TypeDBClient*" exception is much more intuitive than a sort of native exception "RuntimeError". Let's investigate further if we can get down to 1 error to catch (better UX)

Reply:
We introduce a class `TypeDBClientException`, generated by SWIG. Exceptions of this class are returned from the rust layer. All exceptions from the python layer are now instances of `TypeDBClientExceptionExt(TypeDBClientException)`, and at the user side we need to check only one exception type.

Reply:
Amazing